tasks/handle-action-provision-complete.yaml

```diff
@@ -5,12 +5,17 @@
       labels:
         state: started
     spec:
-      vars:
-        current_state: started
+      vars: "{{ vars_update | from_yaml }}"
     status:
       towerJobs:
         provision:
           completeTimestamp: '{{ anarchy_run_timestamp }}'
+  vars:
+    vars_update: |
+      current_state: started
+      {% if 'messages' in anarchy_action_callback_data %}
+      provision_messages: {{ anarchy_action_callback_data.messages | default(provision_messages, true) | to_json }}
+      {% endif %}
```

Comment:
first comment: i'm wondering if the `default(provision_message, true)` will never be executed and thus useless because it's inside the block `if 'messages' in anarchy_action_callback_data`?

Would the following work ?

```yaml
- name: Set state started for {{ anarchy_subject_name }}
  vars:
    vars_update:
      current_state: started
      provision_messages: "{{ anarchy_action_callback_data.messages | default(omit) }}"
  anarchy_subject_update:
    spec:
      vars: "{{ vars_update }}"
...
```

or maybe even:

```yaml
- name: Set state started for {{ anarchy_subject_name }}
  anarchy_subject_update:
    metadata:
      labels:
        state: started
    spec:
      vars:
        current_state: started
      vars:
        current_state: started
        provision_messages: "{{ anarchy_action_callback_data.messages | default(omit) }}"
```